public/index.html

```diff
@@ -6,6 +6,7 @@
     <link href="https://fonts.googleapis.com/css?family=Poppins|Revalia" rel="stylesheet">
     <link rel='stylesheet' type='text/css' href='/css/style.css'>
     <title>hypernews</title>
+    <script src="https://www.gstatic.com/firebasejs/4.2.0/firebase.js"></script>
   </head>
   <body>
     <script src="/bundle.js"></script>
```

Comment:
we can move this to the HEAD section and add a `defer` prop to ensure it loads async 🎉

Reply:
Example:

```html
  <!-- ... -->
  <script defer src="https://www.gstatic.com/firebasejs/4.2.0/firebase.js"></script>
  <script defer src="bundle.js"></script>
</head>
```